cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/AbstractQueueAdmin.java

```diff
@@ -27,7 +27,7 @@
  * Common implementation of table-based QueueAdmin
  */
 public abstract class AbstractQueueAdmin implements QueueAdmin {
-  private final String unqualifiedTableNamePrefix;
+  protected final String unqualifiedTableNamePrefix;
```

Comment:
I left the base class to use String, for levelDb/inmemory, and only HBase uses TableId.